_data/changelogs/component-text-input-accessibility.yml

```diff
@@ -0,0 +1,8 @@
+title: Text input accessibility tests
+type: component
+items:
+  - date: NNNN-NN-NN
+    summary: Added accessibility tests page.
+    affectsGuidance: true
+    githubPr: 2697
```

Comment:
Noticed that the PR numbers in the changelogs are appearing as `uswds-site#NNNN`. Might be having a space cadet moment, but not quite sure why it is doing that. 

![image](https://github.com/uswds/uswds-site/assets/93996430/44d18120-fdd2-4c27-b76e-50e25063bd44)

Reply:
I had used NNNN as a placeholder until I had the PR number... Odd it didn't rebuild when I pushed up the changes 🤔 

I'll see if I can trigger a rebuild to get this updated correctly!

Reply:
This is showing appropriately in the latest builds 👍